app/Http/Controllers/Api/UserController.php

```diff
@@ -20,4 +22,16 @@ public function update(UpdateRequest $request, User $user): JsonResponse
 
         return response()->json(['message' => 'Updated successfully'], Response::HTTP_OK);
     }
+
+    public function list(): JsonResponse
+    {
+        $emails = $this->userService->list();
```

Comment:
the name of the method should describe what this method is for

for ex.: getEmailsList